### PR TITLE
[TIMOB-19753] Fix UserActivity macros

### PR DESCRIPTION
**Pull request by hansemannn**
JIRA: https://jira.appcelerator.org/browse/TIMOB-19753

### CODE REVIEW

File: iphone/Classes/TiAppiOSSearchableItemProxy.h [hunk 1]
```diff
@@ -5,10 +5,10 @@
  * Please see the LICENSE included with this distribution for details.
  */
 #if IS_XCODE_7
+#ifdef USE_TI_APPIOS
```
Comment (cheekiatng) [inline iphone/Classes/TiAppiOSSearchableItemProxy.h]: should this be 
```
#ifdef USE_TIAPPIOSSEARCHABLEITEM
```
instead?

File: iphone/Classes/TiAppiOSSearchableItemProxy.m [hunk 2]
```diff
@@ -5,12 +5,11 @@
  * Please see the LICENSE included with this distribution for details.
  */
 #if IS_XCODE_7
+#ifdef USE_TI_APPIOS
```
Comment (cheekiatng) [inline iphone/Classes/TiAppiOSSearchableItemProxy.m]: should this be 
```
#ifdef USE_TIAPPIOSSEARCHABLEITEM
```
instead?

File: iphone/Classes/TiAppiOSSearchableItemProxy.h (hunk 1 above)
Comment (hansemannn) [inline iphone/Classes/TiAppiOSSearchableItemProxy.h]: The legacy plan was to not replace every constant as part of this PR, but yes it makes sense. Will update the remaining constants. Thanks!

**Comment by cheekiatng** [comment]
APPROVED!

**Pull request merged**